kinder/pkg/cluster/manager/actions/actions.go

```diff
@@ -238,6 +246,25 @@ func ValidateCopyCertsMode(t CopyCertsMode) error {
 	return nil
 }
 
+// SupportedKubeadmConfigVersion returns the list of supported kubeadm config version
+func SupportedKubeadmConfigVersion() []string {
+	return []string{
+		"v1beta1",
+		"v1beta2",
```

Comment:
should this include v1beta3 now? also, should we add a deprecation notice on v1beta1 - assuming we should keep this around for testing older versions -